src/measurement/route/get-measurement.ts

```diff
@@ -15,14 +17,13 @@ const handle = async (ctx: ParameterizedContext<DefaultState, DefaultContext & R
 	const result = await store.getMeasurementString(id);
 
 	if (!result) {
-		ctx.status = 404;
-		return;
+		throw createHttpError(404, `Couldn't find the requested item.`, { type: 'not_found' });
```

Comment:
Why? There's no reason to throw these where they can be set directly, and the extra message doesn't have much value either.

Reply:
This is because of message. It is out of sync with the doc: https://github.com/jsdelivr/globalping/blob/get-rate-limiter/public/v1/components/responses.yaml#L66

Reply:
Maybe lets use `Couldn't find the requested measurement.`? To differentiate missing measurement and wrong path.

Reply:
Ok sure